docs/en/stack/ml/anomaly-detection/ml-ad-explain.asciidoc

```diff
@@ -0,0 +1,117 @@
+[[ml-ad-explain]]
+= Anomaly score explanation
+:keywords: {ml-init}, {stack}, {anomaly-detect}, anomaly score, typical value, \
+actual value
+:description: An explanation of how the anomaly score is calculated for \
+{ml} {anomaly-detect}.
+
+Every anomaly has an anomaly score assigned to it. That score indicates how 
+anomalous the data point is, which makes it possible to define its severity 
+compared to other anomalies. This page gives you a high-level explanation of the 
+critical factors considered for calculating anomaly scores, how the scores 
+are calculated, and how renormalization works.
+
+
+[discrete]
+[[score-impact-factors]]
+== Anomaly score impact factors
+
+Three factors can affect the initial anomaly score of a record: 
+
+* single bucket impact,
+* multi-bucket impact,
+* anomaly characteristics impact.
+
+{anomaly-jobs-cap} split the time series data into time buckets. The data within 
+a bucket is aggregated using functions. Anomaly detection is happening on the 
+bucket values.
+
+
+[discrete]
+[[single-bucket-impact]]
+=== Single bucket impact
+
+The probability of the actual value in the bucket is calculated first. This 
+probability depends on how many similar values were seen in the past. It often 
+relates to the difference between actual and typical values. The typical value 
+is the median value of the probability distribution for the bucket. This 
+probability leads to the single bucket impact. It usually dominates the initial 
+anomaly score of a short spike or dip.
+
+[discrete]
+[[multi-bucket-impact]]
+=== Multi-bucket impact
+
+The probabilities of the values in the current bucket and the preceding 11 
+buckets consist of the multi-bucket impact. The accumulated differences between 
+the actual and typical values result in the multi-bucket impact on the initial 
+anomaly score of the current bucket. High multi-bucket impact indicates unusual 
+behavior in the interval preceding the current bucket, even if the value of this  
+bucket may be within the 95% confidence interval.
+
+Different signs mark the anomalies with high multi-bucket impact to highlight 
+the distinction. A cross sign "+" represents these anomalies in {kib}, instead 
+of a circle.
+
+
+[discrete]
+[[anomaly-characteristics-impact]]
+=== Anomaly characteristics impact
+
+The impact of the anomaly characteristics considers the different features of 
+the anomaly, such as its length and size. The total duration of the anomaly is 
+considered, and not a fixed interval as in the case of the multi-bucket impact 
+calculation. The length might be only one bucket or thirty, but even more. 
```

Comment:
The "but even more" seems a bit unclear to me. Something like this is more straight-forward:
```suggestion
calculation. The length might be only one bucket or thirty (or more) buckets. 
```
It still make me wonder if there is an upper maximum number, but maybe that's too low-level of a detail.